ui/js/controllers/perf/graphs.js

```diff
@@ -762,7 +762,7 @@ perf.controller('GraphsCtrl', [
 
             if ($stateParams.series) {
                 $scope.seriesList = [];
-                if (_.isString($stateParams.series)) {
+                if (typeof ($stateParams.series) === 'string') {
```

Comment:
The parentheses around `$stateParams.series` are not needed, could you remove them?